faunadb-scala/src/test/scala/faunadb/ClientSpec.scala

```diff
@@ -136,7 +135,7 @@ class ClientSpec
   }
 
   override protected def afterAll(): Unit = {
-    testDataCleanup()
+    //testDataCleanup()
   }
```

Comment:
i don't think you need to comment this out.

Reply:
Ahhh...good catch. Fixed.